prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    div{
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
```

Comment:
Rather than target all `div` elements, we want to specifically target the element with the `id` attribute of `"card-deck"`. This can be accomplished with an id selector.

https://learning-fuze.github.io/lfz-onboarding/primers/css/#selectors

```css
#card-deck {

}
```